### DISCUSSION TITLE
Paste Image

### DISCUSSION THREAD

**Discussion by kennygokh** [chhoumann/quickadd#609] [Q&A]
Hi, I am using QuickAdd capture to add daily journal entry and one of the item I always paste screenshot/image. I am not able to get it pasted when quickadd prompt. Is the paste image supported? Please advise.

**Comment by chhoumann** [comment]
Answering extremely late - sorry - but with good news, because this works now.

Pasting an image directly into the QuickAdd prompt window still isn't supported (the prompt is a plain text input). But as of QuickAdd 2.14.0, the `{{CLIPBOARD}}` token understands images in Capture choices: if your clipboard holds an image rather than text, QuickAdd saves it as an attachment (following your normal Obsidian attachment location settings) and inserts an embedded link to it.

So for the journal flow: take the screenshot so it lands on the clipboard, then run a Capture choice whose format includes the token, e.g.

```
- {{VALUE}}
{{CLIPBOARD}}
```

You type the text part in the prompt, and the screenshot gets embedded right below it. One thing to know: clipboard *text* takes precedence, so if you copy text after the screenshot, the token inserts that text instead of the image.

Alternatively, if you'd rather paste by hand, enable the **Open** toggle in the Capture choice's Behavior section - QuickAdd opens the note right after the capture with the cursor at the end of what it inserted, and Obsidian's native paste handles the image from there.

Docs: https://quickadd.obsidian.guide/docs/FormatSyntax#clipboard

**Comment by chhoumann** [comment]
Quick follow-up: the missing piece has now shipped. As of [2.18.0](https://github.com/chhoumann/quickadd/releases/tag/2.18.0) you can paste an image straight into the prompt window - QuickAdd saves it as an attachment (following your normal attachment settings) and inserts the embed link at the cursor. You can mix text and images in one value, and paste several images at once. The `{{CLIPBOARD}}` route above still works, but you no longer need the detour.
